.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,39 @@
+---
+name: 🐞 Bug report
+about: Create a bug report
+title: ''
+labels: ["C-bug"]
+assignees: ''
+
+---
+
+<!--
+Note: Please search to see if an issue already exists for the bug you encountered.
+-->
```

Comment:
Can we also add that people can add as much or as little info as they like, the fields are for help, not mandatory.

I don't want to scare people away from reporting bugs because they don't want to fill all these out.